coderd/database/modelmethods.go

```diff
@@ -226,9 +226,31 @@ func (w Workspace) WorkspaceTable() WorkspaceTable {
 }
 
 func (w Workspace) RBACObject() rbac.Object {
+	// if w.IsPrebuild() {
+	//	return w.AsPrebuild()
+	//}
```

Comment:
Remove commented code

Reply:
Addressed in https://github.com/coder/coder/pull/18333/commits/98576c6fea47a61005c5943ab29dafc270da75d7